Makefile

```diff
@@ -53,6 +53,15 @@ copy-resources:
 	cp training/src/main/resources/com/google/myanmartools/compatibility.tsv clients/ruby/lib/myanmar-tools/resources
 	cp training/src/main/resources/com/google/myanmartools/compatibility.tsv clients/php/resources
 
+	cp genconvert/input/udhr_mya.txt clients/js/resources
```

Comment:
Nit: the other two resource files are sorted by filename: all zawgyiUnicodeModel.dat copies are together, and then all compatibility.tsv copies are together.  Please sort these new copies by filename also.

Reply:
Done